faq/wallets/keepkey/keepkey-terms-of-service.md

```diff
@@ -0,0 +1,95 @@
+
+KeepKey does not guarantee the truthfulness, accuracy or reliability of any User Content or endorse any opinions expressed by you or anyone else. By submitting or posting the User Content you fully and unconditionally release and forever discharge KeepKey and its officers, directors, employees and agents from any and all claims, demands and damages (actual or consequential, direct or indirect), whether now known or unknown, of every kind and nature relating to, arising out of or in any way connected with: (A) disputes between you and one or more users or any other person or entity, or (B) the use by KeepKey or you of the User Content, including, without limitation, any and all claims that use of the User Content pursuant to these Terms violates any of your intellectual property rights, copyrights, rights of publicity or privacy, “moral rights,” or rights of attribution and integrity. You acknowledge and agree that KeepKey has no control over, and shall have no liability for any damages resulting from, the use (including, without limitation, re-publication) or misuse by you or any third party of any User Content. KeepKey acts as a passive conduit for User Content and has no obligation to screen or monitor User Content. If KeepKey becomes aware of any User Content that allegedly may not conform to these Terms, KeepKey may investigate the allegation and determine in its sole discretion whether to take action in accordance with these Terms. KeepKey has no liability or responsibility to Users for performance or nonperformance of such activities.
+
+KeepKey has the absolute right to remove and/or delete without notice any User Content within its control that it deems objectionable. You consent to such removal and/or deletion and waive any claim against KeepKey for such removal and/or deletion. KeepKey is not responsible or liable for failure to store posted content or other materials you transmit through the Service. You should take measures to preserve copies of any data, material, content or information you post on the Service or any other sites or platforms.
+
+1. Our Proprietary Rights
```

Comment:
The numbering from here to "Limitation of Liability" is all "1.".

If there were in a valid list they'd increment, but they are not due to the way this is formatted.

I expect the rendered result is not what we expect.